conf/config.json.template

```diff
@@ -39,10 +39,16 @@
   },
   "aws": {
     "aws_access_key_id": "{{ aws_access_key_id }}",
-    "aws_secret_access_key": "{{ aws_secret_access_key }}"
+    "aws_secret_access_key": "{{ aws_secret_access_key }}",
+    "geo_preference": "eu-west-1"
```

Comment:
@iprotsyuk So far, aws region is specified in Ansible for all components of SM Platform

Reply:
ok, we do need this new parameter
for consistency please rename it to `aws_region`